### PR TITLE
POTUS: Flipping out to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

In ref to: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767 switching out to use the text template

Instant Answer Page: https://duck.co/ia/view/potus

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @numbertheory

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@GuiltyDolphin The tests are failing on this; I've done something stupid and obvious wrong, but I can't see it?

File: t/POTUS.t [hunk 1]
```diff
@@ -8,87 +8,71 @@ use DDG::Test::Goodie;
...
         'Grover Cleveland was the 22nd President of the United States.',
-        structured_answer => {
-            input     => ['22nd'],
-            operation => 'President of the United States',
-            result    => 'Grover Cleveland'
-        }
+        '22nd President of the United States',
+        'Grover Cleveland'
     ),
     'potus 16' => test_zci(
```
Comment (GuiltyDolphin) [inline t/POTUS.t]: You're calling `test_zci` here - but you want `build_test`. (There's some others).

Comment (mintsoft) [inline t/POTUS.t]: @GuiltyDolphin Ah! Stupid me, see I told you it was obvious! :+1:

File: t/POTUS.t [hunk 2]
```diff
@@ -8,90 +8,75 @@ use DDG::Test::Goodie;
...
-    'POTUS 24' => test_zci(
+    'POTUS 24' => build_test(
         'Grover Cleveland was the 24th President of the United States.',
-        structured_answer => {
-            input     => ['24th'],
-            operation => 'President of the United States',
-            result    => 'Grover Cleveland'
-        }
+        'Grover Cleveland',
+        '24th President of the United States',
```
Comment (GuiltyDolphin) [inline t/POTUS.t]: Looks like we can just return the name and position, the rest is static (with an optional fourth flag for the current position).

File: t/POTUS.t [hunk 3]
```diff
@@ -8,90 +8,75 @@ use DDG::Test::Goodie;
...
+        }
+    });
+}
+
 ddg_goodie_test(
     [qw( DDG::Goodie::POTUS)],
-    'who is president of the united states' => test_zci(
+    'who is president of the united states' => build_test(
         'Barack Obama is the 44th President of the United States.',
-        structured_answer => {
```
Comment (GuiltyDolphin) [inline t/POTUS.t]: We might want to use a static time for these, as this can change.

Comment (mintsoft) [inline t/POTUS.t]: Well, at the moment when Trump/Hillary is elected the YAML'll need updating, the test will be updated at the same time

Comment (GuiltyDolphin) [inline t/POTUS.t]: @mintsoft I guess.

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks! :dolphin:

**Pull request merged**